getting-started/index.rst

```diff
@@ -99,6 +166,21 @@ directory that was created earlier when you cloned the source code:
 To download tools, bootstrap will use ``wget``, ``curl`` or
 ``lwp-request`` if installed.  If not, it will give an error telling
 you the URL to download from by hand and where to copy the file to.
+If you wish to prevent bootstrap from downloading and building the autotools
+pass the ``--without-autotools`` option.  You can force it to delete the downloaded
+and installed versions by passing ``--clean``.
+
+Our bootstrap script will check which directories you have checked out,
+so you can bootstrap a partial tree.  You can also ``touch .nobootstrap`` in
+a subdirectory to tell bootstrap to ignore it.
```

Comment:
You can also now list the directories you want on the bootstrap command-line, which is typically more convenient than touching `.nobootstrap`.  They're sticky if bootstrap gets rerun automatically, but not if you rerun it again by hand.

Reply:
👍